### PR TITLE
Better organize landing page

### LINKED ISSUE #2255: Welcome page for trusted organization officials soft-locks user
## Description
As a trusted organization official, I get soft-locked after logging into [app.zetkin.org](https://app.zetkin.org/)

## Steps to reproduce
1. Go to app.zetkin.org
2. login into an trusted organization official account

### Expected Behaviour

There exists UI that the user can interact with, including a link to the users page and a logout button.

### Actual Behaviour

The user is soft-locked as there is nothing they can interact with on the page. There is neither a link to the users page or a logout button.

### How [app.zetkin.org](https://github.com/zetkin/app.zetkin.org) looks like after  trusted organization official login:

Welcome page for trusted organization officials:

- [ ] ❌ Has content
- [ ] ❌  Has a ui the user can interact with
- [ ] ❌  Has a logout button
- [ ] ❌  Has a  link to the user's page

Screenshot 2024-10-12 at 10 28 26

Screenshot 2024-10-12 at 10 26 56

### How Zetk.in looks  like after  trusted organization official login:

For comparison here is how the welcome page for trusted organization officials looks like in  [Zetk.in](https://www.zetk.in/):

-  ✅ Has content
-  ✅ Has a ui the user can interact with
-  ✅ Has a logout button
-  ✅ Has a  link to the user's page

image

### PR DESCRIPTION
**Pull request by richardolsson**
## Description
This PR improves the landing page for organizers, in two ways. First, it adds an error message to explain to non-officials why the page is otherwise empty (the problem described in #2255). Second, it adds a welcome message explaining what Gen3 is, which is in preparation of directing all organizer users to Gen3.

In the process of implemented this, I needed a less wide layout for the alerts to look good, so I changed the layout entirely to a centered, max 800px wide layout.

### When signed in as official  (organizer/admin)
image

### When signed in as non-official users
image

## Changes
* Changes the layout of the /organize landing page to not have a menu
* Replaces the old Zetkin logo with the new one
* Adds an error message when the user has no official memberships
* Adds an info alert about Gen3

## Notes to reviewer
None

## Related issues
Resolves #2255

### CODE REVIEW

**Comment by richardolsson** [comment]
I'm wanting to hear from @sefsh and/or @Bagera if you feel that the look of the new page is good enough to go live.

File: src/features/organizations/l10n/messageIds.ts [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+  gen3: {
+    description: m(
+      'This is the new (generation 3) organizer web app. If you are used to the old one, you will find lots of new features and an improved user interface here. But if you want, you can still use the old organizer app for a while longer.'
+    ),
+    gen2Button: m('Go to the old interface'),
+    title: m('Welcome to the new Zetkin!'),
+  },
+  notOrganizer: {
+    description: m(
+      'This part of Zetkin is only available to users who have been granted organizer access my another organization official. You do not currently have the sufficient level of access. If you believe that this is a mistake, please reach out to your local Zetkin officials or to Zetkin Foundation.'
```
Comment (ziggabyte) [inline src/features/organizations/l10n/messageIds.ts:13]: Very tiny typo: should be "by" instead of "my"

Comment (richardolsson) [reply src/features/organizations/l10n/messageIds.ts:13]: Thank you! Update coming in a minute. 💯

Comment (richardolsson) [reply src/features/organizations/l10n/messageIds.ts]: Fixed in 7b5bbf6044633fbb1c40833c679e4f038d1e8c43. Please review again. 🙏

**Review by ziggabyte — APPROVED** [review]
🦭 of approval!

**Pull request merged**